lib/cc/cli/prepare/quality.rb

```diff
@@ -0,0 +1,59 @@
+require "safe_yaml/load"
+
+module CC
+  module CLI
+    class Prepare
+      class Quality < Command
+        ENGINES_CONFIG = {
+          "complexity-ruby" => {
+            "enabled" => true,
+            "channel" => "beta",
+          },
+          "duplication" => {
+            "enabled" => true,
+            "channel" => "cronopio",
```

Comment:
Since `cronopio` was intentionally random, branch name will probably change in the future. Using a constant ref avoids us having to change the channel name here, and provides an opportunity to use a clearer name - like `QUALITY_CHANNEL` or `DUPLICATION_QUALITY_CHANNEL`?

Don't feel strongly though. up to you!